CHANGELOG.md

```diff
@@ -1,27 +1,26 @@
 # Lotus changelog
 
-# Node and Miner v1.32.0-rc1 / 2024-12-05
+# Node and Miner v1.32.0-rc2 / 2024-12-13
 
-This is the first release candidate of the upcoming MANDATORY Lotus v1.32.0 release, which will deliver the Filecoin network version 25, codenamed “Teep” 🦵.
+This is the second release candidate of the upcoming MANDATORY Lotus v1.32.0 release, which will deliver the Filecoin network version 25, codenamed “Teep”🦵.
```

Comment:
is it MANDATORY now? might we ship this prior to nv25 now?

probably ok if we update this for rc3, I'm not sure i have good wording to change this for now

Reply:
Well technically this release train will become mandatory at some point, once the scope for what is actually going into nv25 is completed.

I can also just remove that sentence, so that it is not so confusing.